src/v2/components/Applications/Detail/index.jsx

```diff
@@ -44,13 +71,17 @@ const TransactionDetail = () => {
     {
       label: 'Nickname',
       hint: '',
-      value: 'Testname',
+      value: accountInfo.data,
     },
     {
       label: 'Description',
       hint: '',
-      value:
-        'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.',
+      value: 'TODO',
+    },
+    {
+      label: 'Time',
```

Comment:
What is Time here? The last time a transaction called this application?

Reply:
Yes exactly! I reused Time since it was a standard header. I'm up for anything, like "last run" or whatever, just let me know!

Reply:
Let's go with "Last Called" for now